### ISSUE TITLE
Formating number as currency in Jet Engine Forms Calculated Field

### ISSUE THREAD

**Issue by mariodev114** [CrocoBlock/suggestions#4378]
The value of calculated field in Jet Engine Forms can't be formated.
![1](https://user-images.githubusercontent.com/72226033/142736284-113530be-44e5-47bc-8a64-cedab4a1f366.png)

As you can see in the image, the total number of the calculation is 1563975€, which is very clunky to read

I want this number to be displayed as 1.563.975€ but there is no option for that.

**Comment by nreljed** [comment]
Grouped in post #4025 

Concerning the jetengine approach with number fields or numeric values in general it doesn't matter where we are

forms, pages, single, etc.
Numeric values must always be accompanied by a possibility of formatting them
numbers are like languages .. we cannot standardize them and say yes ... 12345 is of course .. 12345
no .. it's not possible to see it like that
it doesn't work that way in the real world 😅

**Comment by mariodev114** [comment]
Yeah, is so frustrating not being able to format numbers of meta fields in jet engine...

**Comment by Crocoblock** [comment]
The JetEngine Form module is deprecated. Instead, use JetFormBuilder
https://i.imgur.com/deXA808.png
